src/registrar/tests/test_url_auth.py

```diff
@@ -23,6 +23,7 @@
     "content_type_id": "2",
     "object_id": "3",
     "domain": "whitehouse.gov",
+    "user_pk": "1",
```

Comment:
Why?

Reply:
This test case uses a URL generator. Basically it will try to "inject" variables given their name. Since i wasn't in that hardcoded definition, that test was failing because "user_pk" was not a valid key in this test case